src/profiles/_profileSupportedOptions.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
what do you plan to do with that list? it's not being imported anywhere

Reply:
its being used but wanted to keep it as some official list that can be used for other layers like backend

Reply:
lists gets outdated and need maintenance, I wouldn't do that unless it's strictly required